### PR TITLE
Book-AwesomeES6

### PR DESCRIPTION
**Pull request by rmjspecial**
**Project description**

Add index.html file
Add index.css file
Add index.js file
Add modules file

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @rmjspecial,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a Pull Request with failing linter

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by rmjspecial** [comment]
fix linters error

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+## Author
+
+👤 **Rejoice Evans**
+
+- GitHub: [@githubhandle]( https://github.com/rmjspecial)
+- twitter:  [@twitterhandle](https://twitter.com/rmjspecial2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rejoice-evans-74882122a/)
+
+## 🤝 Contributing
+
```
Comment (danmainah) [inline README.md]: - [ ] [optional] It would be nice to add a link to your issue page, where contributions could be made.

**Review by danmainah — APPROVED** [review]
Hi @rmjspecial, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**